python/pip.bzl

```diff
@@ -262,6 +262,38 @@ def multi_pip_parse(name, default_version, python_versions, python_interpreter_t
         pip_parses = pip_parses,
     )
 
+def _whl_filegroup_impl(ctx):
+    out_dir = ctx.actions.declare_directory(ctx.attr.name)
+    ctx.actions.run_shell(
+        outputs = [out_dir],
+        inputs = [ctx.file.whl],
+        tools = [ctx.executable._get_wheel_records_tool],
+        arguments = [ctx.attr.pattern],
+        command = '{tool} {wheel} "$1" | xargs unzip -q -d {out_dir} {wheel}'.format(
```

Comment:
Right now this is not hermetic as it depends on `xargs` and `unzip`. I think we could in theory do the `unzipping` entirely within python?

Reply:
yep, absolutely. done.